woodwork/tests/accessor/test_table_accessor.py

```diff
@@ -123,19 +132,21 @@ def test_init_accessor_with_schema(sample_df):
 
     schema_df = sample_df.copy()
     schema_df.ww.init(name='test_schema', semantic_tags={'id': 'test_tag'}, index='id')
-    schema = schema_df.ww.schema
+    schema = schema_df.ww._schema
```

Comment:
Really minor point, but this can stay as `schema_df.ww.schema` since we are not using the schema_df schema in any of the assertions that follow.

Reply:
haha yeah I forgot that by passing `schema` through that that'd be the object no matter what. Thanks for pointing this out!